app/views/v2/layout.mustache

```diff
@@ -0,0 +1,137 @@
+{{! Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
When adding more v2 layout templates, I think we should consider slowly also adding v2 golden files.